deepchem/metalearning/tests/test_maml_reload.py

```diff
@@ -2,43 +2,52 @@
 
 import deepchem as dc
 import numpy as np
-import tensorflow as tf
+import pytest
 
+try:
+  import tensorflow as tf
 
-class SineLearner(dc.metalearning.MetaLearner):
+  class SineLearner(dc.metalearning.MetaLearner):
```

Comment:
I think we can define this outside the try/catch block and just raise an exception in the constructor if tf isn't available. This ensures that the user gets a sensible error if TF isn't installed rather than a mysterious class not found issue

Reply:
This class is initialized as part of the test suites and https://github.com/VIGNESHinZONE/deepchem/blob/set-tf/deepchem/metalearning/maml.py MetaLearner class has a dependency on Tensorflow, so it will throw an error.

Reply:
Is the issue the https://github.com/VIGNESHinZONE/deepchem/blob/set-tf/deepchem/metalearning/maml.py#L198 @tf.function annotation throwing an error if tf isn't available?